### ISSUE TITLE
Not all parameters changes require SSPs to be recomputed

### ISSUE THREAD

**Issue by jonathansick** [dfm/python-fsps#1]
I'd suggest changing the `ssps` and `compute` functions in `fsps.f90` to make it clear that not all parameter changes will cause the SSPs to change. You can see how I accomplished this with the function signatures in my original  [fsps.f90](https://github.com/jonathansick/pySPS/blob/master/pysps/fsps.f90):

```
setup_all_ssp(imf, imf1, imf2, imf3, vdmc, mdave, dell, delt, sbss, fbhb, pagb)
subroutine comp_sp(dust, zmet, sfh, tau, const, &
        fburst, tburst, dust_tesc, dust1, dust2, dust_clumps, &
        frac_no_dust, dust_index, mwr, wgp1, wgp2, wgp3, &
        duste_gamma, duste_umin, duste_qpah, tage)
```

This way a person can quickly call `comp_sp` repeatedly without regenerating SSPs if IMF and isochrone modifications aren't being made. An alternative would be giving python-fsps some smarts in knowing if a new set of parameters will require the SSPs to be regenerated, but I haven't looked at that code yet.

The bottom line is that the code structure needs to give the user confidence in knowing when and when not to regenerate SSPs.
